### DISCUSSION TITLE
Would you consider a Mailing list box?

### DISCUSSION THREAD

**Discussion by ikeagaladrim** [CaiJimmy/hugo-theme-stack#278] [Ideas]
Hi,

It'd be great to have an 'on-brand' box to the right or the left of the page where visitors can input their email address.

Something like this;

![image](https://user-images.githubusercontent.com/73709534/127019233-e104b18c-8625-4d8d-9684-ea29538a2065.png)
 
Be even better to have it integrated with CoverKit or Revue - but that's more of a roadmap request!

Sincerely,
a random CaiJimmy coffee buyer 👍

**Comment by CaiJimmy** [comment]
Since Hugo is a static blog, this feature **will require a third-party service integration**, and each one of them might have a different way to integrate and style. And I don't feel that this is a necessary feature for a personal blog theme.

Sorry, but I probably won't implement this, at least for now 😓
